README.md

```diff
@@ -31,68 +32,76 @@
        transcribe  Transcribe one or more audio file(s)
    ```
 
+
 ## Example usage
 
 - A normal real time session using a .wav file as the input audio
 
    ```shell
-   $ URL=ws://realtimeappliance.mycompany.io:9000/v2
-   $ speechmatics transcribe--url $URL --lang en --ssl-mode=none example_audio.wav
+   $ URL=ws://realtimeappliance.yourcompany:9000/v2
```

Comment:
Put it above the examples as as the common part?

Observant people will figure it out yet...

Reply:
Fixed